Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -671,7 +708,6 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
       auto& trackIndex = primVerGIs[ti];
       if (src == GIndex::Source::ITS && mFillTracksITS) {
```

Comment:
The types of different track will be growing, isn't it better to use a "track-sources" option like
https://github.com/AliceO2Group/AliceO2/blob/3d7bf78847055e5e16f41b8ddb88cc60cdd36a71/Detectors/GlobalTrackingWorkflow/src/primary-vertexing-workflow.cxx#L61
https://github.com/AliceO2Group/AliceO2/blob/3d7bf78847055e5e16f41b8ddb88cc60cdd36a71/Detectors/GlobalTrackingWorkflow/src/primary-vertexing-workflow.cxx#L74 ?
Then you can simply check
`if (GIndex::includesSource(src, sourcesMask)) {`

Reply:
Thank you for this advice, I didn't think about it :)